neural_lam/vis.py

```diff
@@ -5,10 +5,13 @@
 
 # Local
 from . import utils
+from .datastore.base import BaseCartesianDatastore
 
 
 @matplotlib.rc_context(utils.fractional_plot_bundle(1))
-def plot_error_map(errors, data_config, title=None, step_length=3):
+def plot_error_map(
+    errors, datastore: BaseCartesianDatastore, title=None, step_length=1
```

Comment:
Since we can now just get `step_length` from the datastore there is no longer any need to have it as a separate argument here.

This is exactly the type of cleanup I hoped to see when introducing this kind of object (datastore) that contains all information about the data setup. Yay for cleaner code :smile:

Reply:
Good idea! :)

I have made this change in https://github.com/leifdenby/neural-lam/commit/9fcf06e5811d5b3c934a1a93f56a626cdb957555